scripts/setup/stripe_connector.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import uuid
+from typing import Dict
+
+import requests
+import setup.constants as constants
+import setup.secrets as secrets
+
+from fides.api.ops.api.v1 import urn_registry as urls
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+STRIPE_KEY = "stripe"
+
+
+def create_stripe_connector(
+    auth_header: Dict[str, str],
+    key: str = STRIPE_KEY,
```

Comment:
I'd preference not using a constant var here as it's confusing for that not to be in `constants.py`

Reply:
That makes sense. Updated!